src/features/unified-share-modal/__tests__/EmailForm.test.js

```diff
@@ -302,6 +306,44 @@ describe('features/unified-share-modal/EmailForm', () => {
             );
         });
 
+        test('should trigger an error and abort submit action when restricted contacts are present and justification is allowed but not selected', () => {
+            const message = 'test message';
+            const event = { preventDefault: jest.fn() };
+            const onSubmit = jest.fn();
+
+            const wrapper = getWrapper({
+                onSubmit,
+                restrictedExternalEmails: [expectedContacts[1].value, expectedContacts[2].value],
+                selectedContacts: expectedContacts,
+                isRestrictionJustificationEnabled: true,
+            });
+
+            wrapper.setState({ message });
+            wrapper.instance().handleSubmit(event);
+
+            expect(wrapper.state('contactsRestrictionError')).toEqual('boxui.unifiedShare.justificationRequiredError');
+            expect(onSubmit).toHaveBeenCalledTimes(0);
+        });
+
+        test('should trigger an error and abort submit action when restricted contacts are present and justification is not allowed', () => {
```

Comment:
Could this test and the previous test be combined into a `test.each` loop?

Reply:
Yes, looks like most of the test's code is the same. Let me update